completions/zsh/_brew

```diff
@@ -47,7 +47,7 @@ __brew_completion_caching_policy() {
   (( $#tmp )) || return 0
 
   # otherwise, invalidate if latest tap index file is missing or newer than cache file
-  tmp=( ${HOMEBREW_REPOSITORY:-/usr/local/Homebrew}/Library/Taps/*/*/.git/index(om[1]N) )
+  tmp=( ${HOMEBREW_REPOSITORY:-${HOMEBREW_PREFIX}}/Library/Taps/*/*/.git/index(om[1]N) )
```

Comment:
```suggestion
  tmp=( ${HOMEBREW_REPOSITORY:-${HOMEBREW_PREFIX}/Homebrew}/Library/Taps/*/*/.git/index(om[1]N) )
```

I believe it should be like this to equal the previous state.

Reply:
Interesting point, thanks.